source/object-model/computed-properties-and-aggregate-data.md

```diff
@@ -1,9 +1,52 @@
+When a computed property depends on the contents of an array, there are a few
+extra methods you'll need to use in order to correctly recognize when the
+contents of the array change. Arrays have two special keys you can append to
+array properties to track changes on them, `[]` and `@each`.
+
+## `[]`
+
+Sometimes a computed property needs to update when items are added or removed from an array.
```

Comment:
It should also trigger if you replace one of the items (like, when you put a value directly in an index).

Reply:
You're right!

```
Sometimes you need a computed property to update when you add, remove, or replace an item in an array.
```
^^ Do you think that's too concise?

I think it'd also be worth calling the replacement case out explicitly in the two lists for `[]` and `@each` as well. 

I'll make a new commit with a crack at the changes and squash once we settle on the final wording

Reply:
Alternate wording
```
Sometimes you need a computed property to update when the membership of an array changes.
```